tests/components/heos/test_config_flow.py

```diff
@@ -120,3 +120,59 @@ async def test_discovery_flow_aborts_already_setup(
 
     assert result["type"] is FlowResultType.ABORT
     assert result["reason"] == "single_instance_allowed"
+
+
+async def test_reconfigure_shows_form(hass: HomeAssistant, config_entry) -> None:
```

Comment:
can just be part of the next test imo

Reply:
Sure, though I'd prefer to keep the tests tighter and focused on one use-case at a time (following the arrange, act, assert pattern).

Reply:
I feel like the next test is basically `reauth_works` and then this would be better placed there (and saves on a bit of code)